assets/js/intlTelInput.min.js

```diff
@@ -0,0 +1,1201 @@
+  /*
```

Comment:
You can add this library also via npm: https://www.npmjs.com/package/intl-tel-input I think it would be more cleaner, what do you think?

Reply:
cool! I try with this